Documentation/project-docs/dogfooding.md

```diff
@@ -204,3 +204,32 @@ which should make things work (but is fragile, confirm file timestamps that you
 The instructions above were only about updates to the binaries that are part of Microsoft.NETCore.App, if you want to test a package
 for library that ships in its own nuget package you can follow the same steps above but instead add a package reference to the
 individual library package from your `bin\packages\Debug` folder.
+
+### Consuming non-NetStandard assets in a .NET Core 2.0 application
```

Comment:
This heading is getting put under the `Using your local CoreFx build` section.  Maybe we should remove one `#` so it gets its own section.